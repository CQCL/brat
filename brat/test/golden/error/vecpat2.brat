fst3(Vec(Int, 3)) -> Option(Int)
fst3(cons(x,xs)) = some(x)
fst3(some(x)) = none
