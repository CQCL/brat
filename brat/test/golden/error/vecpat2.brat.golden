Error in test/golden/error/vecpat2.brat@FC {start = Pos {line = 3, col = 6}, end = Pos {line = 3, col = 13}}:
fst3(some(x)) = none
     ^^^^^^^

  "some" is not a valid constructor for type Vec

